lighthouse-core/audits/deprecations.js

```diff
@@ -33,7 +33,7 @@ class Deprecations extends Audit {
     return {
       category: 'Deprecations',
       name: 'deprecations',
-      description: 'Site does not use deprecated APIs',
+      description: 'Avoids deprecated APIs',
```

Comment:
I'm all for nixing `Site`, but `Avoids` strikes me as vaguer than `Does not use`. I suggest s/Avoid/Does not use/ throughout.